### PR TITLE
CDAP-14468 improve distinct aggregator widget

### PR DESCRIPTION
**Pull request by albertshau**
Makes the fields property a drop down and adds a get schema
button.

### CODE REVIEW

File: core-plugins/widgets/Distinct-batchaggregator.json [hunk 1]
```diff
@@ -1,15 +1,25 @@
...
+          "widget-type": "csv",
           "label": "Fields",
-          "name": "fields"
+          "name": "fields",
+          "widget-attributes": {
+            "delimiter": ",",
+            "value-placeholder": "Field Name"
+          },
+          "plugin-function": {
+            "method": "POST",
```
Comment (CuriousVini) [inline core-plugins/widgets/Distinct-batchaggregator.json:18]: What is the default label for this widget?

File: core-plugins/widgets/Distinct-batchaggregator.json [hunk 2]
```diff
@@ -1,15 +1,25 @@
...
+            "widget": "outputSchema",
+            "output-property": "schema",
+            "plugin-method": "outputSchema"
```
Comment (CuriousVini) [inline core-plugins/widgets/Distinct-batchaggregator.json:21]: do we need `position` and `button-class`?

**Review by CuriousVini — APPROVED** [review]
2 questions about widget json. Code changes lgtm 👍

File: core-plugins/widgets/Distinct-batchaggregator.json (hunk 1 above)
Comment (albertshau) [reply core-plugins/widgets/Distinct-batchaggregator.json:18]: It shows up as 'Get Schema'

File: core-plugins/widgets/Distinct-batchaggregator.json (hunk 2 above)
Comment (albertshau) [reply core-plugins/widgets/Distinct-batchaggregator.json:21]: don't know what those are, but it renders fine, and this is what the group by aggregator does.

**Comment by CuriousVini** [comment]
LGTM 👍

**Pull request merged**